packages/env/src/index.ts

```diff
@@ -40,11 +40,14 @@ if (environment === 'development' || environment === 'test') {
       FILE_PUBLIC_PATH,
       FILES_STORAGE_PATH,
       DEFAULT_PROJECT_ID: '1',
-      DEFAULT_PROVIDER_API_KEY: 'd32da7c2-94fd-49c3-8dca-b57a5c3bbe27',
+      DEFAULT_PROVIDER_API_KEY: '33275751-f0c4-46f3-bc9a-cd2fb22d86ca', // fake
       NEXT_PUBLIC_POSTHOG_KEY: '',
       NEXT_PUBLIC_POSTHOG_HOST: '',
+      DATASET_GENERATOR_WORKSPACE_APIKEY:
+        'e2c21df6-4e2d-4703-9b65-5f4dee0add23', // fake
```

Comment:
`:ecce_homo:` I really hate all this not working by default dev setup

Reply:
Also we should unifiy under 1 unique project `Copilot` all prompts used for suggestions and copilots to simplify the setup and the amount of env vars

Reply:
toy en ello